cmd/editorconfig-checker/main.go

```diff
@@ -64,6 +65,10 @@ func init() {
 
 	currentConfig, _ = config.NewConfig(configPaths)
```

Comment:
Side note, out of this PR consideration

@theoludwig @klaernie why are we ignoring the error here?

Reply:
Same question for `config.Parse()` call a few liner below

Reply:
I've asked myself the same question for #380 and rewrote a bit of it, didn't I? But no matter, that should be a separate issue.

Reply:
Yes, exactly

Reply:
Just read over it again, to be sure: in #380 I started verifying the error returned by `config.Parse()`, since that is the relevant point where reading a custom config file must succeed for us to continue, yet we can safely ignore the error if we tried to parse the default configuration file.